package.json

```diff
@@ -2,6 +2,6 @@
   "name": "tezos-translator",
   "version": "1.0.0",
   "dependencies": {
-    "nearley": "^2.16.0"
+    "nearley": "^2.19.0"
```

Comment:
you have tested that bump, right?

Reply:
I will revert to 2.16, this is an accidental change